app/models/cloud_tenant.rb

```diff
@@ -8,7 +8,6 @@ class CloudTenant < ApplicationRecord
   has_many   :cloud_subnets
   has_many   :network_ports
   has_many   :network_routers
-  has_many   :shared_cloud_networks, -> { where :shared => true }, :through => :ext_management_system, :source => :cloud_networks
```

Comment:
I think best will be to keep cloud_networks as the relation, so I don't have to define class there. That one is also exposed to automate, so it's expected to stay that way, so we don't break backwards compatibility.

But we really need just one alias to 'all_networks', or a better naming. :-)